docs/source/examples/smoothing.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>This should be an easy addition to my PR, so it can be cleaned up with the class-based system.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>